packages/nimbus/src/components/toggle-button/toggle-button.recipe.ts

```diff
@@ -57,13 +57,10 @@ export const toggleButtonRecipe = defineRecipe({
         },
       },
     },
-    tone: {
-      ...buttonRecipe.variants?.tone,
-    },
   },
   defaultVariants: {
     size: "md",
     variant: "outline",
-    tone: "primary",
+    colorPalette: "primary",
```

Comment:
You don't need this to be a `variant`, since it is a styled-system prop and is not a value in this recipe's `variants` array.

Instead, you should be able to declare this on line 7:
```
...buttonRecipe.base,
"colorPalette": "primary"
```